recipe/meta.yaml

```
@@ -10,7 +10,7 @@ source:
 
 build:
   skip: true  # [win and vc<14]
```

To avoid these issues in the future
```suggestion
  skip: true  # [win and vc<14]
  error_overlinking: true
```

---

@isuruf, thanks, I didn't know about that option.